mobile/src/common/enums/ui/app-font-family.enum.ts

```diff
@@ -0,0 +1,8 @@
+enum AppFontFamily {
+  INTER_400 = 'Inter-400',
```

Comment:
Name all fonts files in kebab-case, link them again and check that they are applied for text component